akka-docs/rst/scala/http/client-side/websocket-support.rst

```diff
@@ -3,6 +3,81 @@
 Client-Side WebSocket Support
 =============================
 
-Not yet implemented see 17275_.
+Client side WebSocket support is available through ``Http.singleWebSocketRequest`` ,
+``Http.webSocketClientFlow`` and ``Http.webSocketClientLayer``.
+
+As a WebSocket consists of two streams of messages, incoming messages (a :class:`Sink`) and outgoing messages
+(a :class:`Source`) where either may be signalled first; or even be the only direction in which messages flow during
+the lifetime of the connection. Therefore a websocket connection is modelled as either something you connect a
```

Comment:
s/websocket/WebSocket/ as it is capitalized as this everywhere else in this page.